### DISCUSSION TITLE
Project Reunion 0.8 RC available! 🎉

### DISCUSSION THREAD

**Discussion by andrewleader** [microsoft/WindowsAppSDK#931] [Announcements]
We're getting close to releasing the stable version of 0.8, and have a release candidate available to try! 🎊 This release candidate does NOT contain experimental features, as it is an early candidate for the final release. This release candidate contains several new bug fixes compared to the previous 0.8 Preview.

To learn more about what's new and how to use the 0.8 RC, please see [the 0.8 RC release](https://aka.ms/projectreunion/0.8-release-candidate).

**Comment by riverar** [comment]
I have to ask. Why does a preview release warrant NuGet publish but not a release candidate? Or was this just some one-off technical issue?

**Reply by andrewleader** [reply]
Good question, this was a one-off.

**Comment by mdtauk** [comment]
> We're getting close to releasing the stable version of 0.8, and have a release candidate available to try! 🎊 This release candidate does NOT contain experimental features, as it is an early candidate for the final release. This release candidate contains several new bug fixes compared to the previous 0.8 Preview.
> 
> To learn more about what's new and how to use the 0.8 RC, please see [the 0.8 RC release](https://aka.ms/projectreunion/0.8-release-candidate).

I see ``AcrylicBrush.BackgroundSource`` has been removed.  I'm assuming this is temporary until Background Acrylic is supported after 1.0 (WinUI 3)

Do you have any ETA or progress to share on getting that working and supported @andrewleader ?

**Reply by andrewleader** [reply]
That's correct. I don't think we have any new ETA's or progress to share yet, but when we do, that'll be shared/updated on the WinUI repo: https://github.com/microsoft/microsoft-ui-xaml

**Comment by mrlacey** [comment]
The [list of changes](https://github.com/microsoft/ProjectReunion/releases/tag/v0.8.0-rc) doesn't seem very big. Looking at them in isolation it's hard to imagine them warranting a release of this magnitude. I know it's not a major version bump, but increasing three minor versions was originally explained as a way of indicating how much the project had grown/progressed since the 0.5 release.
Now it doesn't look like a lot has changed in that time. Have I missed something obvious? (Where the small release notes hide big changes.)

All the changes in this (candidate) release appear to be for WinUI3. What progress is happening in the other projects that are part of Project Reunion?

The small number of features and fixes in this release could be seen to suggest that much has been cut from this release as it's not yet ready, but it will hopefully be included in the 1.0 release.
This is always a possibility when you define shipping dates (& version numbers) that have to be met rather than waiting for functionality to be ready. Is anything being done to ensure that features/functionality planned for the 1.0 release won't also be kept back at the last moment and hoped (and planned) for features won't slip to a post-1.0 release?

What (if anything) is now at risk of slipping from the plans for the 1.0 release?

**Reply by andrewleader** [reply]
Thanks Matt. 0.8 was never planned to have any major new features in the supported GA release if you look at the roadmap. A lot of our progress outside of WinUI 3 in the 0.8 timeframe was on preview/experimental support for unpackaged apps (which we shipped a preview of earlier this month).

![image](https://user-images.githubusercontent.com/13246069/122285173-4191d700-cea3-11eb-8263-94f90f0a0504.png)
